tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/AsyncTimeoutTest.java

```diff
@@ -78,7 +79,8 @@ public static WebArchive deploy() {
 
         JavaArchive testJar = ShrinkWrap
                 .create(JavaArchive.class, "ftAsyncTimeout.jar")
-                .addClasses(AsyncTimeoutClient.class, AsyncClassLevelTimeoutClient.class, Connection.class)
+                .addClasses(AsyncTimeoutClient.class, AsyncClassLevelTimeoutClient.class, Connection.class,
+                        TestException.class)
```

Comment:
Unsure why `TestException` is needed in this app, none of the other classes in the app appear to reference it.